src/components/Selector/Selector.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import './Selector.css';
+
+export default function Selector({ setHead, setMiddle, setBottom, catchphrase, setCatchphrase }) {
+  return (
+    <div className="left">
+      <div className="picker">
+        <label>
+          Head
+          <select id="head-dropdown" onChange={(e) => setHead(e.target.value)}>
+            <option value="bird">Bird</option>
+            <option value="duck">Duck</option>
+            <option value="dog">Dog</option>
+            <option value="horse">Horse</option>
+          </select>
+        </label>
+        <label>
+          Middle
+          <select id="middle-dropdown" onChange={(e) => setMiddle(e.target.value)}>
+            <option value="blue">Blue</option>
+            <option value="dress">Fancy</option>
+            <option value="pink">Pink</option>
+            <option value="red">Red</option>
+          </select>
+        </label>
+        <label>
+          Bottom
+          <select id="bottom-dropdown" onChange={(e) => setBottom(e.target.value)}>
+            <option value="leg">Single Leg</option>
+            <option value="white">White Pants</option>
+            <option value="blue">Blue Jeans</option>
+          </select>
```

Comment:
Awesome work with your onChange handler functions Alex, hope this lab felt good to complete!